python/dolfinx/wrappers/io.cpp

```diff
@@ -31,9 +31,13 @@ namespace dolfinx_wrappers
 void io(py::module& m)
 {
 
+  // dolfinx::io::cell vtk cell type converter
+  m.def("get_vtk_cell_type", &dolfinx::io::cells::get_vtk_cell_type);
+
   // dolfinx::io::cell permutation functions
   m.def("perm_vtk", &dolfinx::io::cells::perm_vtk);
   m.def("perm_gmsh", &dolfinx::io::cells::perm_gmsh);
+  m.def("transpose_map", &dolfinx::io::cells::transpose);
```

Comment:
This is a super cryptic function name. C++ is marginally better (since it's in an extra namespace).

I'd suggest removing it from Python and performing the operation locally. Avoid bloating the Python API will niche functions that can be easily done from Python.